ITEST/itest.storelogdata/test-resources/simple3_Hello.req.json

```diff
@@ -0,0 +1,3 @@
+{
+  "hour": 5
+}

```

Comment:
Replace with a constant string in the test. Like `REQUEST="{\r\n  \"hour\": 5\r\n}\r\n"`
Do it for the all new added files.